src/runtime/pkg/govmm/qemu/qemu.go

```diff
@@ -433,6 +442,10 @@ func (object Object) QemuParams(config *Config) []string {
 		deviceParams = append(deviceParams, string(object.Driver))
 		deviceParams = append(deviceParams, fmt.Sprintf("id=%s", object.DeviceID))
 		deviceParams = append(deviceParams, fmt.Sprintf("host-path=%s", object.File))
+	case RMEGuest:
+		objectParams = append(objectParams, string(object.Type))
+		objectParams = append(objectParams, fmt.Sprintf("id=%s", object.ID))
```

Comment:
Hi @kevinzs2048 . Is there any API for CCA to set `PersonalizationValue` for qemu? This can help to enable initdata. If any, the extra code can be very simple
```golang
if len(object.InitdataDigest) > 0 {
			// PersonalizationValue in Arm-CCA should be exactly 64 bytes
			personalizationValueSlice := adjustProperLength(object.InitdataDigest, 64)
			hostdata := base64.StdEncoding.EncodeToString(personalizationValueSlice)
			objectParams = append(objectParams, fmt.Sprintf("<the parameter of pv for CCA>=%s", hostdata))
		}
```

Reply:
Thanks @Xynnn007, Good point, let me update the PersonalizationValue.

Reply:
Done, sorry a bit late due to biz travelling last week, please take time to review, thanks! @Xynnn007